Answers.md

```diff
@@ -1,7 +1,16 @@
 # Questions
 
-1. What is the DOM?
-2. What is an event?
+1. What is the DOM? 
+The DOM stands for Document Object Model which is a tree-like representation of a webpages contents.
+
+2. What is an event? 
+Events are actions that occur on the webpage like mouse-clicks, scrolling or keypresses.
+
 3. What is an event listener?
+Event listener is a function that waits for an event to occur.
+
 4. Why would we convert a NodeList into an Array?
+The DOM is not Javascript native. You would convert a NodeList into an Array if you wanted to use the methods array.forEach() , array.filter() , etc...
```

Comment:
I would give the example of array.map() instead of .forEach, because .forEach() actually works with NodeLists :+1: